ledger/metrics/src/main/scala/com/daml/metrics/HttpJsonApiMetrics.scala

```diff
@@ -25,43 +30,76 @@ class HttpJsonApiMetrics(override val prefix: MetricName, override val registry:
 
   val surrogateTemplateIdCache = new CacheMetrics(prefix :+ "surrogate_tpid_cache", registry)
 
+  object DropWizardMetrics extends DropwizardFactory {
```

Comment:
To improve readability, we might want to simply name this as a factory:
```suggestion
  val unlabeledMetricsFactory = new DropwizardFactory {
```